test/controllers/user.controller.test.js

```diff
@@ -70,6 +70,31 @@ describe('User Controller', () => {
             });
         });
       });
+
+      context('when user registers as a vendor ', () => {
+        it('returns successful token', (done) => {
+          const vendor = UserFactory.build({ vendor: { companyName: 'Highrachy Investment' } });
+          request()
+            .post('/api/v1/user/register')
+            .send(vendor)
+            .end((err, res) => {
+              expect(res).to.have.status(201);
+              expect(res.body.success).to.be.eql(true);
+              expect(res.body.message).to.be.eql('User registered');
```

Comment:
I think we should update this:

```suggestion
              expect(res.body.message).to.be.eql('Company registered');
```

Reply:
my approach to this causes an error tho